documentation/sandbox-guide/movies.adoc

```diff
@@ -0,0 +1,315 @@
+
+will return all kinds of nodes (limiting to 20 items).
+
+
+== Properties
+
+Properties are name-value pairs that are used to add attributes to nodes and relationships.
+
+To return specific properties of a node you can write -
+
+[source, cypher]
+MATCH (m:Movie) return m.title, m.released
+
+image::{images}/movies-properties.jpg[width="100%"]
+
+*Expected Result* - This will return Movie nodes but with only the `title` and `released` properties.
+
+*Try*
+
+1. Wrtie a query to get `name` and `born` properties of the Person node.
++
+[source.try-cypher-result,cypher]
+MATCH (p:Person) return p.name, p.born
+
+== Create a Node
+
+`Create` clause can be used to create a new node or a relationship. 
+
+[source, cypher]
+Create (p:Person {name: 'John Doe'}) RETURN p
+
+The above statement will create a new `Person` node with property `name` having value `John Doe`.
+
+*Try*
+
+1. Create a new `Person` node with a property `name` having the value of your name.
++
+[source.try-cypher-result,cypher]
+Create (p:Person {name: '<Your Name>'}) RETURN p
+
+== Finding Nodes with *Match* and *Where* Clause
+
+`Match` clause is used to find nodes that match a particular pattern. This is the primary way of getting data from a Neo4j database. 
+
+In most cases, a `Match` is used along with certain conditions to narrow down the result.
+
+[source, cypher]
+Match (p:Person {name: 'Tom Hanks'}) RETURN p
+
+This is one way of doing it. Although you can only do basic string match based filtering this way (without using `WHERE` clause).
+
+Another way would be to use a `WHERE` clause which allows for more complex filtering including `>`, `<`, `Starts With`, `Ends With`, etc
+
+[source, cypher]
+MATCH (p:Person) where p.name = "Tom Hanks" RETURN p
+
+Both of the above queries will return the same results.
+
+You can read more about Where clause and list of all filters here - https://neo4j.com/docs/cypher-manual/4.1/clauses/where/
+
+*Try*
+
+1. Find the movie with title "Cloud Atlas"
++
+[source.try-cypher-result,cypher]
+MATCH (m:Movie {title: "Cloud Atlas"}) return m
+
+2. Get all the movies that were released between 2010 and 2015.
++
+[source.try-cypher-result,cypher]
+MATCH (m:Movie) where m.released > 2010 and m.released < 2015 RETURN m
+
+== Merge Clause
+
+The `Merge` clause is used to either
+
+1. match the existing nodes and bind them or
+2. create new node(s) and bind them
+
+It is a combination of `Match` and `Create` and additionally allows to specify additional actions if the data was matched or created.
+
+[source, cypher]
+MERGE (p:Person {name: 'John Doe'})
+ON MATCH SET p.lastLoggedInAt = timestamp()
+ON CREATE SET p.createdAt = timestamp()
+Return p
+
+The above statement will create the Person node if it does not exist. If the node already exists, then it will set the property `lastLoggedInAt` to the current timestamp. If node did not exist and was newly created instead, then it will set the `createdAt` property to the current timestamp.

```

Comment:
```suggestion
The above statement will create the Person node if it does not exist. If the node already exists, then it will set the property `lastLoggedInAt` to the current timestamp. If the node did not exist and was newly created instead, then it will set the `createdAt` property to the current timestamp.

```